addons/website/static/tests/tours/website_update_column_count.js

```diff
@@ -188,10 +188,14 @@ registerWebsitePreviewTour("website_mobile_order_with_drag_and_drop", {
     {
         content: "Click on the second column of the 'Columns' snippet",
         trigger: `${columnsSnippetRow} > div:nth-child(2)`,
-        run: "click",
+        run() {
+            const el = this.anchor;
+            el.scrollIntoView();
+            el.click();
+        },
```

Comment:
@shsa-odoo What if we remove this run function and in next step we change the trigger to: ```trigger: "body .o_overlay_options .o_move_handle, body .o_overlay_options .o_move_handle:not(:visible)",```